k9mail-library/src/main/java/com/fsck/k9/mail/TransportProvider.java

```diff
@@ -16,12 +16,12 @@ public static TransportProvider getInstance() {
         return transportProvider;
     }
 
-    public synchronized Transport getTransport(Context context, StoreConfig storeConfig)
+    public synchronized Transport getTransport(Context context, StoreConfig storeConfig, boolean shouldHideHostname)
```

Comment:
I don't like the idea of passing this as additional constructor argument. The flag should probably go into `StoreConfig`.

Reply:
I didn't like it either.

Is StoreConfig only created from the URL or do we already feed other stuff into it.

Reply:
`StoreConfig` is an interface that `Account` implements. You were probably thinking of `ServerSettings`.